.github/workflows/build-ffmpeg.yml

```diff
@@ -551,3 +553,79 @@ jobs:
           branch: ffmpeg-${{ matrix.os_type }}-${{ matrix.arch }}
           force: false
           message: update-${{ steps.date.outputs.date }}
+
+  update_sunshine:
+    # todo - add if logic to only run when branches were updated
+    if: >-
+      startsWith(github.repository, 'LizardByte/')
+    # &&
+    # ${{ github.ref == 'refs/heads/master' && github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    # todo - enable needs
+    # needs: [ffmpeg]
+    env:
+      update_branch: update/ffmpeg-submodules
+      submodule_dir: third-party
+    steps:
+
+      - name: Checkout Sunshine
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository_owner }}/Sunshine
+          ref: nightly
+          submodules: recursive
+          persist-credentials: false  # otherwise, the token used is the GITHUB_TOKEN, instead of the personal token
+          fetch-depth: 0  # otherwise, will fail to push refs to dest repo
+
+      - name: Update submodules
+        run: |
+          # list of submodules
+          SUBMODULES=$(cat <<- VAREOF
+            ffmpeg-linux-aarch64
+            ffmpeg-linux-x86_64
+            ffmpeg-macos-aarch64
+            ffmpeg-macos-x86_64
+            ffmpeg-windows-x86_64
+          VAREOF
+          )
+
+          echo "::group::git checkout"
+          git checkout -b ${{ env.update_branch }} || git checkout ${{ env.update_branch }}
```

Comment:
how to properly rebase nightly onto this branch without adding additional commits?

Reply:
`git rebase origin/nightly` ?

Reply:
If you can check locally, `git rebase -i origin/nightly` will interactively show the commits (which you need to confirm before proceeding).